### ISSUE TITLE
BCU doesnt detect the vmware installer

### ISSUE THREAD

**Issue by djbanana** [Klocman/Bulk-Crap-Uninstaller#443]
the program , does not detect vmware player in the list of installed programs , the location of vmware player is in the x86 program files folder, Im running windows 11

**Comment by Klocman** [comment]
Does it appear in the `Programs and Features` list?

**Comment by DevDorrejo** [comment]
> Does it appear in the `Programs and Features` list?

yes, appear in 'Programs and Features' list, i am having the same issue, windows 11 22h2 Enterprise.

![image](https://user-images.githubusercontent.com/7697566/231039868-6fbbd781-1b69-42e5-bf16-0ee72509f296.png)

**Comment by DevDorrejo** [comment]
Ok, enabling the checking of 'show protected elements it shows the app in the list.

**Comment by shaojs321** [comment]
> 好的，选中“显示受保护的元素”框，它将在列表中显示该应用程序，但不允许卸载它。

I have the same problem, did you solve it?

**Comment by Klocman** [comment]
> Ok, checking the box 'show protected elements it shows the app in the list., but not allow uninstall it.

Disable protection in `Settings > Uninstallation`.

**Comment by NekoJonez** [comment]
I think this issue should be renamed to "VMWare uninstaller is only shown when you show protected elements" no?
